Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -678,6 +697,7 @@ static void webkit_settings_class_init(WebKitSettingsClass* klass)
             _("Whether to load site icons ignoring image load setting."),
             FALSE,
             readWriteConstructParamFlags);
```

Comment:
```suggestion
            static_cast<GParamFlags>(readWriteConstructParamFlags | G_PARAM_DEPRECATED));
```

Reply:
If you're in the mood for a drive-by cleanup, this is missing from `enable-java` too. Might as well fix that.